### DISCUSSION TITLE
.NET Maui Splashscreen Background Issue

### DISCUSSION THREAD

**Discussion by Systellence** [dotnet/maui#14054] [Q&A]
Hello!

Not sure if this could be a bug or what

If I set the splashscreen color it only works if the theme is Light. For the Dark theme it stays black. You see the image with a white background though. This is now for Android.

MauiSplashScreen Include="Resources\Splash\logo.svg" Color="White"

I would like the image and the background to be white in Dark or Light theme.

Thanks!

Jaco

**Comment by shadowfoxish** [comment]
That may be a decent feature request, assuming the underlying platforms actually allow for it. Being in the csproj file, it seems to be something that gets compiled, or at least translated into something the underlying platform would understand during build time.

Suggestion; If your app has a color theme, why not use your accent color for the background of the splash? That would make it pop/be visually interesting, and not look like you're disrespecting a user's theme preference.

**Comment by Systellence** [comment]
Hello

The issue was with API 31. A known issue apparently. Yes I want to honour the theme colour but later. First testing it for Light.

Kind regards

Jaco Pretorius
